doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -169,11 +175,11 @@ Traceability to Reference Model.
 | `req.sec.zon.04` | Zoning | The Architecture **must** support integration with Corporate Identity Management systems. |
 | `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable standards and regulations. |
 | `req.sec.cmp.03` | Compliance | The Architecture **must** comply with all applicable regional standards and regulations. |
-| `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with ISO 27001 and adhere to the V1.1 NIST Cybersecurity Framework. |
-| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with ISO27001. |
+| `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. |
+| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. |
```

Comment:
What applicable means? The requirement should not be open ended.

Reply:
"applicable" ---> relevant. As it is not possible to list all of the security standards by each region (state) and country, we need a generic statement. However, an example of an ISO standard that is used quite widely (referred to in EU as well as US standards) is listed as an example. 

Please propose alternate language, if the current language is unacceptable.

Reply:
Issue #649 for further discussion on this topic.

Reply:
I would propose to comment out these requirements and add them back in #649

Reply:
@pgoyal01 I do not see these as commented out. Do I overlook something?

Reply:
@CsatariGergely Done.